addon/styles/components/choose-many.scss

```diff
@@ -25,12 +25,12 @@
   padding: 15px;
   box-sizing: border-box;
   align-self: center;
-  background-color: $lighter-gray;
-  color: $dark-gray;
+  background-color: var(--cs-input-background);
+  color: var(--cs-input-copy);
   text-align: center;
 
   &:hover {
-    background-color: $light-gray;
+    background-color: var(--cs-input-background-hover);
```

Comment:
Here we don't need to set `background-color` property again. We can just change the value of the css variable. `cs-input-background: var(--cs-input-background-hover);`. Same for `--cs-input-copy`.

But since we may not want to change a global variable like that, you can use a local variable and define it at the top for this component so that we also know which variables are later going to change. Something like:

```
.cs-component-choose-one {
  --cs-component-choose-one-background: var(--cs-input-background);
  --cs-component-choose-one-copy: var(--cs-input-copy);
}
```

Then:
```
.cs-component-choose-one--label {
  background-color: var(--cs-component-choose-one-background);

  &:hover {
    --cs-component-choose-one-background: var(--cs-input-background-hover);
 }
}

.cs-component-choose-one--label.checked {
  --cs-component-choose-one-background: var(--cs-input-selected);
  --cs-component-choose-one-copy: white;
}
```

I'm trying out the suggestions in this [guide](https://www.smashingmagazine.com/2018/05/css-custom-properties-strategy-guide/) ("Change The Value Not The Variable") to see if it will benefit us in the long term.

Reply:
Interesting read, I didn't realize variables could be scoped like that. CSS variables make more sense to me now :) I'll make the changes as you suggested.